### DISCUSSION TITLE
Implement checkbox questions in MathObjects

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/pg#746] [Ideas]
It would be nice to have a MathObjects version of checkbox multiple choice questions, akin to parserRadioButtons but forcing students to select all correct answers.

**Comment by somiaj** [comment]
@dlglin I already have a macro that does this. This macro allows both radio buttons and checkboxes, and even has a partial credit option where the check boxes can be treated as a series of T/F questions, in which students get partial credit for which they correctly check and correctly don't check.

https://github.com/somiaj/pgMacros/blob/main/parserMultipleChoice.pl

This also better saves the the info in the answer box, saving the actual label (vs the index) used, as you mentioned in #708.

This has worked well for all of my needs, but may need some improvements before being added to pg. I also wanted to update how options are sent to it, as I don't like how I set it up, but haven't wanted to change it since I have others who are using it in their problems.

**Reply by taniwallach** [reply]
https://github.com/openwebwork/pg/pull/767 seems to have been merged into develop to provide such functionality.
